examples/quickstarts/httpd.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/httpd-ex/blob/master/README.md.",
   "labels": {
-    "template": "httpd-example"
+    "template": "httpd-example",
+    "app": "httpd-example"
```

Comment:
The app label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.